package.json

```diff
@@ -337,7 +337,7 @@
     "tsconfig-paths": "4.2.0",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32859&commitId=dc04f8818c00453394352c071b99b05159ea21ae&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|5| Changed|
|Avatar Converged.size+inactive+badge.chromium.png|1| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1167| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2113| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>